config/jobs/kubernetes-sigs/image-builder/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
list taken from https://github.com/kubernetes-sigs/image-builder/blob/master/OWNERS (alphabetized)

Reply:
We should update them in https://github.com/kubernetes-sigs/image-builder/blob/master/OWNERS first since that's the source of truth. cc @codenrhoden